cilium-dbg/cmd/policy_import.go

```diff
@@ -40,7 +42,12 @@ var policyImportCmd = &cobra.Command{
 
 			for _, r := range ruleList {
 				if err := r.Sanitize(); err != nil {
-					Fatalf("%s", err)
+					if errors.Is(err, api.ErrFromToNodesRequiresNodeSelectorOption) {
+						// Don't error out as this can't be validated client-side
+						fmt.Printf("Validation of policy %s has been skipped as it can't be validated.\n", path)
```

Comment:
nit: I think the current wording of this (and the other similar message) implies that no validation occurs, but from a user's perspective, that's not entirely true. I think that a message like this would be more clear to the user:

```suggestion
						fmt.Printf("Validation of policy %s has been skipped in the client, further validation will occur server-side.\n", path)
```